packages/kbn-optimizer/limits.yml

```diff
@@ -87,6 +87,7 @@ pageLoadAssetSize:
   licenseManagement: 41817
   licensing: 29004
   lists: 22900
+  logsShared: 281060
```

Comment:
Have we looked into loading parts of this plugin async?

Reply:
I lazy loaded all the components I can, but most of the exported logic lives in utilities and state machines imported by the infra plugin, so the bundle size gain is minimum :(